Mathlib/NumberTheory/NumberField/Units/Regulator.lean

```diff
@@ -287,15 +295,55 @@ theorem regulator_eq_det (w' : InfinitePlace K) (e : {w // w ≠ w'} ≃ Fin (ra
 
 open scoped Classical in
 /--
-The degree of `K` times the regulator of `K` is equal to the absolute value of the determinant of
-the matrix whose columns are `(mult w * log w (fundSystem K i))_i, w` and the column `(mult w)_w`.
+The degree of `K` times the regulator of the family `u` is equal to the absolute value of the
+determinant of the matrix whose columns are
+`(mult w * log w (fundSystem K i))_i, w` and the column `(mult w)_w`.
```

Comment:
Why did you change the docstring? The old one seems to be a precise description. (And there is no "family `u`" in sight here.)
Were you planning to generalize the statement?

Reply:
The statement is indeed generalized above in the file but this change was a pure error on my part